AG-13051 Set focusVisible to false when clicking the chart

olegat opened the pull request:
https://ag-grid.atlassian.net/browse/AG-13051

olegat committed:
AG-13051 Set focusVisible to false when clicking the chart

Also add this to the E2E test.

alantreadway reviewed (approved):
LGTM, but snapshots need to be reviewed?

olegat commented:
> LGTM, but snapshots need to be reviewed?

The new snapshots indicate a defect. I'll need to investigate that because there shouldn't be any snapshot changes for this PR.

olegat committed:
AG-13051 Fix compilation after git rebase

olegat committed:
AG-13051 Simplify KeyNavManager a bit

olegat committed:
AG-13051 Move undo/redo input logic to KeyNavManager

The advantage of this is that the KeyNavManager has access to
previousInputDevice and can automatically decide whether or not the focus
indicator needs to be shown or not. This fixes an e2e test where the focus
indicator is shown after undo/redo key presses.

olegat committed:
AG-13051 Fix undo step of test "toolbar › line"

olegat committed:
AG-13051 Fix redo step of test "toolbar › line"

olegat committed:
nx format

alantreadway reviewed (approved):
LGTM
